crates/rolldown/src/stages/link_stage/mod.rs

```diff
@@ -530,14 +530,14 @@ impl<'a> LinkStage<'a> {
           match reference_ref {
             rolldown_common::SymbolOrMemberExprRef::Symbol(sym_ref) => {
               let canonical_ref = self.symbols.canonical_ref_for(*sym_ref);
-              meta.dependencies.push(canonical_ref.owner);
+              meta.dependencies.insert(canonical_ref.owner);
```

Comment:
If use `Vec` dependencies may have multiple same `ModuleIdx`, it will call multiple `include_module` in tree shaking phase, https://github.com/rolldown/rolldown/blob/8d315d7e0c352658577c648515afc091fa6c6e88/crates/rolldown/src/stages/link_stage/tree_shaking.rs#L64-L73